### PR TITLE
libzip upgrade to 1.5.2

### PR DESCRIPTION
**Pull request by PeterPetrik**
fixes: https://github.com/lutraconsulting/input/issues/17

### CODE REVIEW

File: recipes/qgis/recipe.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ DEPS_qgis=(gdal qca libspatialite libspatialindex expat gsl postgresql libzip qt
 # DEPS_qgis=()
 
 # url of the package
-URL_qgis=https://github.com/qgis/QGIS/archive/fe39b26684dbc7438d7a155421dc0f1af045eae8.tar.gz
+URL_qgis=https://github.com/qgis/QGIS/archive/b5459794f8b877a6d1064fd504b7a3ccb3aaba9a.tar.gz
 
 # md5 of the package
 MD5_qgis=6f17047e9123f0365288339babe64e92
```
Comment (m-kuhn) [inline recipes/qgis/recipe.sh]: Does that work without updating the md5?

**Comment by m-kuhn** [comment]
Whee, nice!

File: recipes/qgis/recipe.sh (hunk 1 above)
Comment (PeterPetrik) [reply recipes/qgis/recipe.sh]: yes it does, but I think that random commit after FF is better than random commit before FF. But I can revert if you like

**Comment by m-kuhn** [comment]
Are there upgrades in QGIS which are required too or is it "just because"?

**Comment by PeterPetrik** [comment]
just in case

**Comment by PeterPetrik** [comment]
hmm, it falls with docker build, let me fix it and get back

**Comment by m-kuhn** [comment]
Thanks, let me know, I have the finger ready to build and deploy.

**Comment by PeterPetrik** [comment]
ready now

**Comment by PeterPetrik** [comment]
confirmed, it works for qgz files in input now

**Comment by m-kuhn** [comment]
Confirmed, works here too, thanks a lot

released as qfield-sdk:20190528

**Pull request merged**